net-snmp/Dockerfile

```diff
@@ -19,6 +19,6 @@ COPY tmpfiles.template service.template manifest.json config.json.template /expo
 RUN mkdir -p /exports/hostfs/etc/{sysconfig,snmp} /exports/hostfs/var/lib/net-snmp/ && \
     cp /etc/sysconfig/snmpd /etc/sysconfig/snmptrapd /exports/hostfs/etc/sysconfig/ && \
     cp /etc/snmp/snmpd.conf /etc/snmp/snmptrapd.conf /exports/hostfs/etc/snmp/ && \
-    echo "# WAIT_FOR_CONFIGURATION=0" >> /exports/hostfs/etc/sysconfig/snmpd
+    echo "NET_SNMP_PRESTART_HOOK='net-snmp-create-v3-user -a password -x encyption-password admin-user'" >> /exports/hostfs/etc/sysconfig/snmpd
```

Comment:
I kind of like it. I think we still end up needing a general snmp for client usage but we could split that work up.

One addition that I think would be helpful is if we rename it from ``NET_SNMP_PRESTART_HOOK`` to ``NET_SNMP_RUN_ONCE`` or something like that. Then after it's executed in the start script the line in ``/etc/sysconfig/snmpd`` would be blanked so it only executes once. WDYT?